Makefile

```diff
@@ -27,16 +27,36 @@ upload:
 
 TAG=$(shell git describe --always)
 
-build/linux/amd64:
+# Individual build targets for each OS/arch combination
+bin/code-marketplace-mac-amd64: $(wildcard **/*.go) go.mod go.sum
```

Comment:
Does `wildcard **/*.go` work? I can't find any example of this online, seems like people do a lot of workarounds for this.

In `coder/coder` we define a variable with the result of a `$(shell find ...)` command